sdk/recoveryservicesbackup/azure-resourcemanager-recoveryservicesbackup/src/test/java/com/azure/resourcemanager/recoveryservicesbackup/RecoveryServicesBackupManagerTests.java

```diff
@@ -0,0 +1,224 @@
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.US_WEST2;
+    private String resourceGroupName = "rg" + randomPadding();
+    private RecoveryServicesBackupManager recoveryServicesBackupManager;
+    private RecoveryServicesManager recoveryServicesManager;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        recoveryServicesManager = RecoveryServicesManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        recoveryServicesBackupManager = RecoveryServicesBackupManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    public void testCreateProtectionPolicy() {
+        Vault vault = null;
+        ProtectionPolicyResource protectionPolicyResource = null;
+        String randomPadding = randomPadding();
+        try {
+            String vaultName = "vault" + randomPadding;
+            String policyName = "policy" + randomPadding;
+
+            // @embedmeStart
+            List<SubProtectionPolicy> lstSubProtectionPolicy = Arrays.asList(
+                new SubProtectionPolicy()
+                    .withPolicyType(PolicyType.FULL)
+                    .withSchedulePolicy(
+                        new SimpleSchedulePolicy()
+                            .withScheduleRunFrequency(ScheduleRunType.WEEKLY)
+                            .withScheduleRunDays(Arrays.asList(DayOfWeek.SUNDAY, DayOfWeek.TUESDAY))
+                            .withScheduleRunTimes(Arrays.asList(OffsetDateTime.parse("2018-01-24T10:00:00Z"))))
```

Comment:
Curiosity only.

Why don't service complain when schedule time is 3 years before?

Reply:
These codes are excerpted from `ProtectionPoliciesCreateOrUpdateSamples# createOrUpdateFullAzureWorkloadProtectionPolicy`, which may not be verified it in the backend. Now the schedule time has been fixed in the new version.

Reply:
Would it work for e.g. `OffsetDateTime.now().plusDays(1)` (one day in future)?

Reply:
Tried this way of setting, but the following exception occurred: 
![image](https://github.com/Azure/azure-sdk-for-java/assets/74638143/b977cd43-23fb-46f1-a941-410f08429236)

Reply:
OK. If this does not work, I am fine with current code.

Though personally my guess is that there is certain aspect in the datetime rfc3339 that service won't parse, either the microsecond, or the timezone.
Wondering whether this would work `OffsetDateTime.now().withNano(0).withOffsetSameInstant(ZoneOffset.UTC)` (i.e. remove the microsecond and timezone to UTC).

Reply:
I've tried this and found that only the following date formats work with `yyyy-mm-ddThh:mm:ssZ`。

Reply:
I assume `OffsetDateTime.now().withNano(0).withOffsetSameInstant(ZoneOffset.UTC)` would give you `yyyy-mm-ddThh:mm:ssZ`?

Reply:
I found the rule of `DateTime`, now fixed in the new version.